OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -212,6 +212,15 @@ public AssetBrowserLogic(Widget widget, Action onExit, World world)
 				prevButton.IsVisible = () => !isVideoLoaded;
 			}
 
+			var node = logic.Value.Nodes.FirstOrDefault(x => x.Key == "SupportedFormats");
+			if (node != null)
+			{
+				var value = node.Value.Value.Replace(" ", "");
+				allowedExtensions = value.Split(',');
+			}
+			else
+				allowedExtensions = new string[0];
```

Comment:
I'd prefer that we defaulted to an empty list, and had everything defined in the yaml.  All the values are highly mod-specific, and this is now the only place that they are hardcoded (everything else goes via mod.yaml).